File: codesite/python_problems/utils/in_place_utils.cpp
#include <iostream>
#include <tuple>
using namespace std;

template<typename Method, typename Inputs>
void runTests(
   Solution& solution,
   Method method,
   Inputs& inputs)
{
   for (auto& data : inputs) {
      apply(
         [&](auto&... args) {
            (solution.*method)(args...);
         },
         data
      );

      apply(
         [&](const auto&... args) {
            (println(args), ...);
         },
         data
      );
   }
}

File: codesite/python_problems/utils/in_place_utils.py
import copy

def run_tests(method):
    for input_data in inputs_list:
        data_copy = copy.deepcopy(input_data)
        # method = getattr(solution, method_name)
        method(*data_copy)
        print(json.dumps(*data_copy))


# run_tests(solution.moveZeroes)
